src/client/page.ts

```diff
@@ -699,10 +699,15 @@ export class BindingCall extends ChannelOwner<channels.BindingCallChannel, chann
   }
 }
 
+function trimEnd(s: string): string {
+  if (s.length > 50)
+    s = s.substring(0, 50) + '\u2026';
```

Comment:
```suggestion
    s = s.substring(0, 50) + '…';
```
I think using the direct symbol makes it directly clear which symbol it is.